### ISSUE TITLE
why different hyperparameters are used?

### ISSUE THREAD

**Issue by dedoogong** [lolrudy/GPV_Pose#4]
Hello! thanks for your kind sharing!
I saw the difference between your implementation and paper regarding the hyperparams(lambda 1, ~, 8 and so on)

flags.DEFINE_float('rot_1_w', 8.0, '')
flags.DEFINE_float('rot_2_w', 8.0, '')
flags.DEFINE_float('rot_regular', 4.0, '')
flags.DEFINE_float('tran_w', 8.0, '')
flags.DEFINE_float('size_w', 8.0, '')
flags.DEFINE_float('recon_w', 8.0, '')
flags.DEFINE_float('r_con_w', 1.0, '')

flags.DEFINE_float('recon_n_w', 3.0, 'normal estimation loss')
flags.DEFINE_float('recon_d_w', 3.0, 'dis estimation loss')
flags.DEFINE_float('recon_v_w', 1.0, 'voting loss weight')
flags.DEFINE_float('recon_s_w', 0.3, 'point sampling loss weight, important')
flags.DEFINE_float('recon_f_w', 1.0, 'confidence loss')
flags.DEFINE_float('recon_bb_r_w', 1.0, 'bbox r loss')
flags.DEFINE_float('recon_bb_t_w', 1.0, 'bbox t loss')
flags.DEFINE_float('recon_bb_s_w', 1.0, 'bbox s loss')
flags.DEFINE_float('recon_bb_self_w', 1.0, 'bb self')

flags.DEFINE_float('mask_w', 1.0, 'obj_mask_loss')

flags.DEFINE_float('geo_p_w', 1.0, 'geo point mathcing loss')
flags.DEFINE_float('geo_s_w', 10.0, 'geo symmetry loss')
flags.DEFINE_float('geo_f_w', 0.1, 'geo face loss, face must be consistent with the point cloud')

flags.DEFINE_float('prop_pm_w', 2.0, '')
flags.DEFINE_float('prop_sym_w', 1.0, 'importtannt for symmetric objects, can do point aug along reflection plane')
flags.DEFINE_float('prop_r_reg_w', 1.0, 'rot confidence must be sum to 1')

those are different from the paper( 1/8.0, 1/8.0, 1/8.0, 1. ,,,. 8.0, 1.0, 1.0) and also it seems like there are ssome unseen, newly added ones like recon_n_w recon_d_w recon_s_w prop_pm_w. 

Could you please give me some advice about it?

Plus, I think I should add 'Geo_face' loss term for 3.5 Bounding Box - Pose Geometric Consistency. Right?

Thank you very much!

**Comment by lolrudy** [comment]
We select the best configuration of hyper-parameters to achieve best performance, so some of them are different from the original setting in the paper. (however, most of them remain the same)
